src/branch_view.js

```diff
@@ -4,8 +4,8 @@ goog.require('utils');
 goog.require('banner_css');
 goog.require('safejson');
 
-function renderHtmlBlob(parent, html, bannerElement) {
-	var re, match;
+function renderHtmlBlob(parent, html, hasApp) {
+	var re, match, ctaText;
```

Comment:
In case the metadata script doesn't exist, you might want to set defaults here:

`var re, match, ctaText = hasApp ? 'OPEN' : 'GET'`